client-ts/Microsoft.AspNetCore.SignalR.Client.TS/HttpConnection.ts

```diff
@@ -4,7 +4,7 @@ import { ITransport, TransferMode, TransportType, WebSocketTransport, ServerSent
 import { IHttpClient, HttpClient } from "./HttpClient"
 import { IHttpConnectionOptions } from "./IHttpConnectionOptions"
 import { ILogger, LogLevel } from "./ILogger"
-import { NullLogger } from "./Loggers"
+import { NullLogger, ConsoleLogger, LoggerFactory } from "./Loggers"
```

Comment:
Do you need to import ConsoleLogger here?

Reply:
probably not - I started cleaning some imports/exports but decided it did not belong in this PR and must have missed it.